src/content/docs/en/guides/content-collections.mdx

```diff
@@ -395,6 +395,16 @@ const publishedBlogEntries = await getCollection('blog', ({ data }) => {
 });
 ```
 
+You can also create draft pages that are available when running the dev server, but not built in production:
+
+```js
+// Example: Filter out content entries with `draft: true` only when building for production
+import { getCollection } from 'astro:content';
+const blogEntries = await getCollection('blog', ({ data }) => {
+  return import.meta.env.PROD ? data.draft !== true : true;
+});
+```
```

Comment:
Note: this example is similar to the one above, but I decided to split it out so it specifically mentions the word "draft pages" which helps with searching, and because the one above has a nice example that doesn't mix well if I combine them 🤔 

I'm open however if you think we should combine them still.